src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: rgba(0, 172, 220, 1);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
+}
+
+.header {
+  /* border: 1px solid red; */
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header img {
+  display: block;
+}
+
+.nav-bar {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-bar li {
+  /* border: 1px solid green; */
+  list-style: none;
+}
+
+.nav-list-elem a {
+  display: inline-block;
+  height: 60px;
+  line-height: 61px;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.margin-left-20 {
+  margin-left: 20px;
```

Comment:
Avoid creating classes that describe the style or spacing directly, such as '.margin-left-20'. It's better to use classes that describe the purpose or context where the spacing is applied.